index.js

```diff
@@ -3,31 +3,39 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const app = express();
 const port = 3000;
-const fs= require('fs');
+const fs = require('fs');
 const data = require('./todo.json');
 const { v4: uuidv4 } = require('uuid');
 
 
 app.use(bodyParser.json())
 
-
-
-app.post('/todos', (req, res) => {
-
-    const { title,description,completed } = req.body
-
-    const titles = data.map(d => d.title);
-
-    if (!titles.includes(title)) {
-        data.push({ title,description,completed, id: uuidv4()});
-        fs.writeFileSync("todo.json",JSON.stringify(data,null,2))
-
-        return  res.send(data)
+app.delete('/todos', (req, res) => {
+
+    const { id, title, description, completed } = req.body
```

Comment:
you dont need title, description, completed , you need only id, please delete all unused parameters